etc/catalog/memory.properties

```diff
@@ -0,0 +1 @@
+connector.name=memory
```

Comment:
I tried to use the *memory* connector to test multiple statement transactions but it does not look to support them.